hlint.cabal

```diff
@@ -126,3 +133,19 @@ executable hlint
     ghc-options:        -rtsopts
     if flag(threaded)
         ghc-options:    -threaded
+
+executable cc-engine
```

Comment:
Could we roll this executable into Hlint - eg have "Hlint cc" do the necessary magic?

Reply:
Assuming we do need a separate executable, would it make more sense to have cc/hlint-cc.cabal? I don't see that many benefits to keeping it in the main one.

Reply:
I thought it was cleaner to keep a separate executable, and only build it on a flag. IMO, users (and developers) of `hlint` directly shouldn't really be exposed to or have to think about the CC cruft daty-to-day. Had I thought of it, I would've done `cc/engine.cabal`, so I'm in favor of that.

That said, if you prefer `hlint cc` just let me know, I see now reason that wouldn't work.